terraform/aws/components/lab-results/locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  default_tags = {
+    Project = var.project
+    Environment = var.environment
+    Component = var.component
+  }
+
+  resource_prefix = "${var.project}-${var.environment}-${var.component}"
+
+  availability_zones = ["${var.region}a", "${var.region}b", "${var.region}c"]
+
+  image_name = "${var.account_id}.dkr.ecr.${var.region}.amazonaws.com/lab-results:${var.lab-results_build_id}"
```

Comment:
I don't know if you have the repo already created, or are you already building images. But you can create and manage the repo with terraform. The repos are controlled by account component, and defined in etc/eu-west-2_account.tfvars, add a new entry there just like for other adapters, apply it and you will have the repo,

Reply:
holding this change for the moment.